workspaces/refresh_liferay_sample_workspace.sh

```diff
@@ -115,6 +115,60 @@ EOF
 
 	mkdir -p liferay-sample-custom-element-2/src/common/components
 
+	cat <<EOF > liferay-sample-custom-element-2/src/common/components/Comic.js
+import React from 'react';
+
+class Comic extends React.Component {
```

Comment:
I didn't think the refresh script was required anymore for projects that aren't generated from somewhere else?

For example, we re-gen projects that rely on create-react-app, but for our own hand created samples(like theme-spritemap) we don't add that to refresh script.

Reply:
yeah i need to revisit this with brian and see if he is ok with this not being generated.

Reply:
Actually this app is still generated with create_react_app.  see here: https://github.com/liferay/liferay-portal/blob/master/workspaces/refresh_liferay_sample_workspace.sh#L114